### ISSUE TITLE
The orange color used for flat buttons and other clickable elements does not meet color contrast standards

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#229]
![Screen Shot 2019-08-28 at 1 07 30 PM](https://user-images.githubusercontent.com/3613392/63877134-f5576900-c994-11e9-91c6-fa9734d83c6d.png)

Use the [WebAIM Color Contrast Checker](https://webaim.org/resources/contrastchecker/) and see that the ratio with white is about 3 to 1 when it will need to be 4.5 to 1 if this will be used for smaller text.

The color in question:
```
#FA6423
```

**Comment by jonambas** [comment]
Closing - no more orange buttons in hibana
